src/main/java/org/mifos/connector/mpesa/utility/MpesaUtils.java

```diff
@@ -0,0 +1,112 @@
+package org.mifos.connector.mpesa.utility;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+
+
+@Component
+public class MpesaUtils {
+
+    @Autowired
+    private MpesaAMSProp mpesaAMSProp;
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private List<MpesaProps.MPESA> mpesa;
+
+    private String process = "process";
+
+    @PostConstruct
+    public String getProcess(){
+        return process;
+    }
+    @PostConstruct
```

Comment:
You should not use PostConstruct for getters. Use only 1 post construct per class.